### ISSUE TITLE
Non-crafting Professions/Feats

### ISSUE THREAD

**Issue by Judicioux** [Tirocupidus/TheExiledRPOverhaul#153]
I've heard some light talk of this in the past, and I'm unsure if it's been officially filed with the GitHub, but here goes. It may be a strong draw to the server and marketing point if we were to somehow implement feats that offer incentives other than crafting recipes.

As has been discussed before, the idea that immediately comes to mind is taking the buff that thrall dancers have and introducing it as a feat for players. Creating that feat, in my opinion, could inspire players to create characters that are not combat or crafting-minded. That would be a very beneficial aspect, I think, of RP that isn't fighting or violence. Something based less on survival and more on entertainment.

I would hope to bring home the potential increase in new players this kind of implementation could create, as it would be something _very_ unique to The Exiled that would inspire an even greater communal environment.

We could move beyond trade (without undermining it, of course) and enable the player base to create a stronger social community within their towns and walls.

**Comment by CraftsmasterKubilai** [comment]
AFAIK, it's not possible for one player to affect another player as described... YET. I'd personally love to make a Bard-type when this sort of thing is possible (and when we can finally get a "play instrument" emote...). May be a while till FC adds that though. =)

**Comment by CupcakeValkyrie** [comment]
Tiro and I have already discussed the possibility of some combat-related feats. The idea being that someone focused _professionally_ on being a soldier would have less time to practice crafting professions. I'm all for expanding the profession tree as far as it can go, if for no other reason than to create a divide between warriors and craftspeople, as currently, you can be both a grandmaster craftsman _and_ the best warrior at the same time.

**Comment by CraftsmasterKubilai** [comment]
Hey now... warrior/smiths are part of the canon... well, in the MOVIE< anyway. =P

**Comment by CupcakeValkyrie** [comment]
Well, first, the movie's not considered canon =P
Second, you can be a warrior/smith...but you just won't be the absolute grandmaster of both.

**Comment by Judicioux** [comment]
I one-hundred percent love and agree with the other ideas we could potentially do with non-crafting feats. I know when I spoke to someone about it a while ago, there was the discussion of bards, of soldiers, of dancers, of thieves. Just, in general, if this sort of concept is possible, it would really enrich the RP, I feel.

**Comment by Judicioux** [comment]
An added benefit of it is that it would increase the value of crafters, especially those with higher tier professions like grandmaster and master.

**Comment by ShiraSelene** [comment]
It's a little bit delayed, but I did want to share some thoughts about non-crafting and non-combat options! 

Firstly, just from an in-character perspective, I think many of our exiles, while still concerned with simple survival, wouldn't mind a few more diverting pastimes--people like to be entertained, after all. Beyond that, though, I think such features would accommodate role-play even more than the game itself does already.

 If dancers, or even musicians, somehow become accessible, The Exiled could potentially draw in a demographic that's gone untouched for a quite while. Whether former players of Star Wars Galaxies, or of other games with bards, there are people out there in search of gameplay that's still compelling, but not necessarily all about fighting and trading. 

In short, I think options of this nature could really make us stand out even more so amongst all of the other servers! :P

**Comment by CraftsmasterKubilai** [comment]
FunCom has said in the past that they plan to add some sort of music/musician mechanic to go with dancers... we might have to wait for them to add that mechanic, but I definitely agree that entertainer is a great idea for a profession... ESPECIALLY if it can be used to make buffing/debuffing Bards (which aren't really present in the source material, but ARE fun to play). =)

I suspect that Sorcery will open up a LOT of different professions when it's finally introduced - everything from Healers to Illusionists to (I hope) summoner types will likely be possible, either vanilla or via mods by manipulating whatever source mechanic they employ to allow one player to change another player's state at-range w/o needing to hit them with a weapon.
